hawkbit-autoconfigure/src/main/java/org/eclipse/hawkbit/autoconfigure/security/SecurityManagedConfiguration.java

```diff
@@ -719,6 +722,20 @@ protected void configure(final HttpSecurity http) throws Exception {
                     .logoutSuccessHandler(logoutSuccessHandler);
         }
 
+        @Bean
+        public HttpFirewall httpFirewall() {
+            final List<String> allowedHostNames = hawkbitSecurityProperties.getAllowedHostNames();
+            final StrictHttpFirewall firewall = new StrictHttpFirewall();
+
+            if (allowedHostNames != null && !CollectionUtils.isEmpty(allowedHostNames)) {
+                firewall.setAllowedHostnames(hostName -> {
+                    LOG.info("Firewall check host: {}, allowed: {}", hostName, allowedHostNames.contains(hostName));
```

Comment:
I would set the log level to debug and extract the `allowedHostNames.contains(hostName)` to local variable and return it as well